internal/gatewayapi/testdata/tlsroute-with-backend.out.yaml

```diff
@@ -121,8 +121,6 @@ xdsIR:
       routes:
       - destination:
           name: tlsroute/default/tlsroute-1/rule/-1
-          settings:
```

Comment:
curious what happened here, and why this got deleted ?

Reply:
In this scenario, the route generally shouldn’t be operational. I’ll look into why the settings are being removed to better understand the problem. For now, the route is left as-is because fully invalidating it causes the conformance test to fail. Perhaps we can ignore the TLS route translation error and create a separate task to address it.